spec/support/corpus.rb

```diff
@@ -80,21 +82,11 @@ def verify_mutation_generation
           start:        method(:start),
           in_processes: parallel_processes
         }
-        total = Parallel.map(effective_ruby_paths, options) do |path|
-          count = 0
-          node =
-            begin
-              Parser::CurrentRuby.parse(path.read)
-            rescue EncodingError, ArgumentError
-              nil # Make rubocop happy
-            end
-
-          if node
-            count += Mutant::Mutator::REGISTRY.call(node).length
-          end
 
-          count
+        total = Parallel.map(effective_ruby_paths, options) do |path|
+          filter_exceptions(path, DEFAULT_MUTATION_COUNT) { count_mutations(path) }
```

Comment:
I notice your intention is to make the `filter_exceptions` method generic, but as long we do not have a 2nd call side, lets simply go for `count_mutations(path)` here that contains the semantics for exception filtering expectation (maybe through another extracted method, but this one should know the defaults by itself).